src/js/server/lib/password.js

```diff
@@ -17,9 +14,9 @@ var crypto = require("crypto");
 fluid.registerNamespace("fluid.express.user.password");
 fluid.express.user.password.encode = function (password, salt, iterations, keyLength, digest) {
     // Set defaults that are useful in dealing with CouchDB and express-couchUser data.
-    iterations = iterations || 10;
-    keyLength  = keyLength  || 20;
-    digest     = digest     || "sha1"; // Already used when the value is omitted, but specified for future-proofing.
+    iterations = iterations !== undefined ? iterations : 10;
+    keyLength  = keyLength !== undefined ? keyLength : 20;
+    digest     = digest !== undefined ? digest : "sha256";
```

Comment:
keyLength doesn't show up in the user record, is that intentional or does it need to be there as well?

Reply:
It doesn't.  You can tell the length from the key itself.  Take a look at all of the test records, which don't have it, but can still be used to log in.

Reply:
In other words, you need to know how long to make the key before you make it, once you make it, you can just see how long it is.

Reply:
@the-t-in-rtf thanks for the clarification, that makes sense.